opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/CommonGivenStages.java

```diff
@@ -117,7 +121,8 @@ public SELF rest_assured_points_to_opba_server_with_fintech_signer_on_banking_ap
     }
 
     public SELF user_registered_in_opba_with_credentials(String username, String password) {
-        RestAssured
+        if (!userCreated) {
```

Comment:
Check for userCreated is insufficient - as such scenario will fail
`user_registered_in_opba_with_credentials('foo', 'bar')`
`user_registered_in_opba_with_credentials('joe', 'bar')`
, can we avoid this check ?

Reply:
changed to set of users